crates/turborepo-ui/src/tui/pane.rs

```diff
@@ -59,6 +49,13 @@ impl<W> TerminalPane<W> {
         Ok(())
     }
 
+    pub fn hassss_stdin(&self, task: &str) -> bool {
+        self.tasks
+            .get(task)
+            .map(|task| task.stdin.is_some())
+            .unwrap_or_default()
+    }
```

Comment:
~~I think this can get lifted to `App` since that's where the data resides.~~ Already was, we can just can this defn and the one below it.